src/backend/web/graphql/index.js

```diff
@@ -94,7 +123,26 @@ module.exports.resolvers = {
         const last = first + prPage < numOFPosts ? first + prPage : numOFPosts;
         const postIds = await getPosts(first, last);
 
-        return Promise.all(postIds.map(id => Post.byId(id)));
+        const result = await Promise.all(postIds.map(id => Post.byId(id)));
+
+        if (filter) {
+          // check if author name is equal to what we're searching for
+          if (filter.author) {
+            return result.filter(post => post.author === filter.author);
+          }
+          // check if published date is between two provided dates
+          if (filter.fromDate || filter.toDate) {
+            const fromDate = filter.fromDate ? new Date(filter.fromDate * 1000) : new Date();
+            const toDate = filter.toDate ? new Date(filter.toDate * 1000) : new Date();
+            return result.filter(post => post.published >= fromDate && post.published <= toDate);
```

Comment:
So you're not using the stuff I landed for stats yet.  Are you going to move toward that?